jina/types/arrays/document.py

```diff
@@ -164,11 +166,11 @@ def reverse(self):
             self._docs_proto.reverse()
 
     @cached_property
-    def _docs_map(self):
-        """Returns a doc_id to doc mapping so one can later index a Document using doc_id as string key.
+    def _id_to_index(self):
```

Comment:
are we sure this is to br substituted, isnt it compatible? where is the docsmap used now?

Reply:
`docs_map` is used in `__index__` to enable via-`id`-access https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md#access-element

Reply:
I don't understand the question. 

I didn't see where else `_docs_map` was used. The interface is respected, as defined by the existing tests

Reply:
ah ok it is private and is used smartly. Ok good for me

Reply:
can I get it approved then?